src/west/manifest.py

```diff
@@ -645,7 +650,9 @@ def __init__(self, path=None, revision='(not set)', url='(not set)',
         self.path = path or self.name
         '''Relative path to the project in the installation.'''
 
-        self.abspath = os.path.realpath(os.path.join(util.west_topdir(),
+        self.west_top = util.west_topdir()
```

Comment:
I think this is a great idea, but could you do it from the Manifest class instead of ManifestProject?

That will make it easier to make it a kwarg to the manifest constructor and factory methods, which we can then feed in from WestCommand later to make this even better.